templates/pages/cart.html

```diff
@@ -23,7 +23,7 @@ <h1 class="p-common__title">{{lang 'cart.title'}}</h1>
             {{> components/cart/content}}
         </div>
 
-        <div class="js-cart-totals p-cart__totals">
+        <div class="js-cart-totals c-totals">
```

Comment:
Ideally, it would be separated into a component named "totals" based on its name - especially if you are reusing in in multiple places. 

Also you already have `cart` component, so you could use "c-cart__totals" instead if it's more like a "one time" component